src/ocamlorg_frontend/pages/blog.eml

```diff
@@ -8,8 +8,8 @@ Layout.render
     <div class="container-fluid wide">
         <h1 class="sr-only">OCaml Blog</h1>
 
-        <div class="grid grid-cols-1 md:grid-cols-3 gap-16">
-            <div class="col-span-2">
+        <div class="grid grid-cols-1 md:grid-cols-3  ">
+            <div class="col-span-2 ">
```

Comment:
```suggestion
        <div class="grid grid-cols-1 md:grid-cols-3">
            <div class="col-span-2">
```

Reply:
Alright, i am still struggling to get where the news data comes from, so that i can reduce the number of news to be displayed

Reply:
check in `src/ocamlorg_web/lib/handler.ml`